.devcontainer/Dockerfile

```diff
@@ -78,10 +56,27 @@ RUN wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/nul
     apt-get update -qq && export DEBIAN_FRONTEND=noninteractive && \
     apt-get install -y --no-install-recommends cmake cmake-curses-gui
 
-# Install editors
+# Install python
 RUN apt-get update -qq && export DEBIAN_FRONTEND=noninteractive && \
     apt-get install -y --no-install-recommends \
-        neovim emacs nano
+        python3 python3-pip
+
+# Install cmake format package
+RUN python3 -m pip install --upgrade pip cmake_format
+
+# Install conan
+RUN python3 -m pip install --upgrade pip setuptools && \
```

Comment:
Why are we upgrading pip twice?

Reply:
The same reason I do apt-get --update before every command. To get the most recent packages. If you want to change the line 68 4 months later you want to get eh latest package.

Reply:
Pip is a program, not a package index. Old versions of pip can still retrieve the same package index that new ones can.

I do not see the point in splitting this up in this way. Why not just `pip install --upgrade pip setuptools conan cmake_format`?

If you want to split these up, it should be:
```bash
pip install --upgrade pip setuptools
pip install conan
pip install cmake_format
```
... but I do not see any advantage to doing it this way.

Reply:
Yes right I will install all python packages in one command